### DISCUSSION TITLE
Submarine weapons

### DISCUSSION THREAD

**Discussion by ElementShadow** [FakeFishGames/Barotrauma#7807] [Ideas and suggestions]
Hello, its me newbie again, wanted to ask as captain i most of time want to change basic weapons on my submarine, and if i know properly its not possible to swap weapons between slots on bases, and also not possible to sell them after changing, i dont understand why its made like this

Isnt it would be better to let people swap them and sell starting ones you dont need for 33% or something -_-)

**Comment by MaaiG** [comment]
> its not possible to swap weapons between slots on bases...
> Isnt it would be better to let people swap them and sell starting ones you dont need for 33% or something -_-)

...just like this https://github.com/Regalis11/Barotrauma/issues/7825
